src/Components/PresentationalComponents/DownloadReportKebab/BaseKebab.js

```diff
@@ -18,7 +18,7 @@ const BaseKebab = ({ dropdownItems, ...props }) => {
         <div>
             <Dropdown
                 onSelect={handleOnKebabSelect}
-                toggle={<KebabToggle onToggle={handleOnKebabToggle} />}
+                toggle={<KebabToggle onToggle={handleOnKebabToggle} isDisabled={props.disabled} />}
```

Comment:
Do you really have to pass the prop `disabled`? you can pass the `{ ...props }` 
`toggle={<KebabToggle onToggle={handleOnKebabToggle} { ...props } />}`

Reply:
@leSamo great catch for the reserved keyword `isDisabled` :D